### PR TITLE
Update deployment.md

### PR DESCRIPTION
**Pull request by tracker1**
Add some references for ARR and running node apps as a service in windows.

### CODE REVIEW

File: deployment.md [hunk 1]
```diff
@@ -3,6 +3,21 @@
...
+* [node-cluster-service](https://github.com/godaddy/node-cluster-service)
+
+### Deployment on Windows
+* Running As A Service
+  * [windows-service](https://www.npmjs.com/package/windows-service)
+  * [node-windows](https://github.com/coreybutler/node-windows)
+  * [NSSM (Non-Sucking Service Manager)](http://nssm.cc/)
+  * [WinSer](http://jfromaniello.github.com/winser/) wrapper for NSSM
+  * Service Alternative: [Use Task Sheduler to run a task at startup](http://www.howtogeek.com/138159/how-to-enable-programs-and-custom-scripts-to-run-at-boot/)
+* [ARR])http://www.iis.net/downloads/microsoft/application-request-routing) (Application Request Routing) for IIS
```
Comment (mousetraps) [inline deployment.md]: For acronyms, let's do name first, and acronym in parentheses:
So [Application Requrest Routing (ARR)](http://www.iis.net/downloads/microsoft/application-request-routing), and same for NSSM above

File: deployment.md [hunk 2]
```diff
@@ -3,6 +3,21 @@
 * Continuous integration with VSO.
 * Docker and containers.
 * Cross-platform remote debugging.
+
+### Keeping Applications Running
+* [PM2](http://pm2.keymetrics.io/)
+* [forever](https://www.npmjs.com/package/forever)
+* [node-cluster-service](https://github.com/godaddy/node-cluster-service)
+
```
Comment (mousetraps) [inline deployment.md]: Any chance you could add some short descriptions here? Or at least one catch all description at the top describing the value add / scenarios in which you'd use something like this?

**Comment by mousetraps** [comment]
Thanks for the PR! Minor comments, and lgtm after that!

File: deployment.md (hunk 2 above)
Comment (romuloctba) [inline deployment.md]: Nodemon could be placed here as well (as it is alone right now (see #27 ))

**Comment by tracker1** [comment]
@mousetraps done.

**Comment by mousetraps** [comment]
:+1:

**Pull request merged**